executors/cmdexec/cmdexec.go

```diff
@@ -35,6 +35,7 @@ type RemoteCommandTransport interface {
 	LVChunkSize() string
 	XfsSw() int
 	XfsSu() int
+	IsContainerizedGluster() bool
```

Comment:
I think we should change this in favor of pulling the exact "wrapper command" string directly from the config. This way heketi does not have to be "smart" about containerized gluster or not, it would be part of the deployment and users could even create their own wrapper scripts if needed.